UKHomeOffice/cerberus-service
# COP-8641: Add tests to verify Selectors grouping with global reference ID

## Description
Add tests to verify Selectors grouping with global reference ID
Scenario : 
Scenario 1: Multiple selectors in the same selector group
GIVEN a task has multiple selectors with the same Group reference ID
WHEN I view the task details
THEN the Group reference details are displayed in the Task details
AND the multiple selectors with the same group reference are displayed under this

## To Test
npm run cypress:runner 

run the following tests from spec `task-details.spec.js`

`Should verify the selector matches with same group reference`
`Should verify the selector matches with same & different group reference on 2 different version of a task`

## Developer Checklist

\* Required
